internal/chart/testdata/charts/dashboard-nginx.yaml

```diff
@@ -0,0 +1,378 @@
+            - name: private-registry-secret
+---
+# Source: dashboard/templates/deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: dashboard-web-4
+    theketch.io/app-name: dashboard
+    theketch.io/app-process: web
+    theketch.io/app-process-replicas: "3"
+    theketch.io/app-deployment-version: "4"
+    theketch.io/is-isolated-run: "false"
+    theketch.io/test-label-all: test-label-value-all
+  name: dashboard-web-4
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: dashboard-web-4
+      theketch.io/app-name: dashboard
+      theketch.io/app-process: web
+      theketch.io/app-deployment-version: "4"
+      theketch.io/is-isolated-run: "false"
+  template:
+    metadata:
+      labels:
+        app: dashboard-web-4
+        theketch.io/app-name: dashboard
+        theketch.io/app-process: web
+        theketch.io/app-deployment-version: "4"
+        theketch.io/is-isolated-run: "false"
+    spec:
+      containers:
+        - name: dashboard-web-4
+          command: ["python"]
+          env:
+            - name: port
+              value: "9091"
+            - name: PORT
+              value: "9091"
+            - name: PORT_web
+              value: "9091"
+            - name: VAR
+              value: VALUE
+          image: shipasoftware/go-app:v2
+          ports:
+          - containerPort: 9091
+      imagePullSecrets:
+            - name: default-image-pull-secret
+---
+# Source: dashboard/templates/deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: dashboard-worker-4
+    theketch.io/app-name: dashboard
+    theketch.io/app-process: worker
+    theketch.io/app-process-replicas: "1"
+    theketch.io/app-deployment-version: "4"
+    theketch.io/is-isolated-run: "false"
+    theketch.io/test-label-all: test-label-value-all
+  name: dashboard-worker-4
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dashboard-worker-4
+      theketch.io/app-name: dashboard
+      theketch.io/app-process: worker
+      theketch.io/app-deployment-version: "4"
+      theketch.io/is-isolated-run: "false"
+  template:
+    metadata:
+      labels:
+        app: dashboard-worker-4
+        theketch.io/app-name: dashboard
+        theketch.io/app-process: worker
+        theketch.io/app-deployment-version: "4"
+        theketch.io/is-isolated-run: "false"
+    spec:
+      containers:
+        - name: dashboard-worker-4
+          command: ["celery"]
+          env:
+            - name: port
+              value: "9091"
+            - name: PORT
+              value: "9091"
+            - name: PORT_worker
+              value: "9091"
+            - name: VAR
+              value: VALUE
+          image: shipasoftware/go-app:v2
+          ports:
+          - containerPort: 9091
+      imagePullSecrets:
+            - name: default-image-pull-secret
+---
+# Source: dashboard/templates/ingress.yaml
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
it should be v1 instead v1beta1
https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122